README.md

```diff
@@ -38,7 +36,7 @@ This driver is based on [the cql-rb gem](https://github.com/iconara/cql-rb) by [
 
 This driver works exclusively with the Cassandra Query Language v3 (CQL3) and Cassandra's native protocol. The current version works with:
 
-* Apache Cassandra versions 2.1, 2.2, and 3.x
+* Apache Cassandra versions 2.1, 2.2, 3.0+, and 4.0+
```

Comment:
I'm wondering if we shouldn't make this 4.0 explicitly (rather than 4.0+).  This driver hasn't been tested against 4.1.x at all and I'm not sure how it will behave against 5.0.

Reply:
Changed 4.0+ to 4.0.x